jest.config.js

```diff
@@ -44,6 +44,6 @@ module.exports = {
       '<rootDir>/config/jest/image.js'
   },
   setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
-  setupFiles: ['raf/polyfill'],
+  // setupFiles: ['raf/polyfill'],
```

Comment:
if not needed let's remove it

Reply:
Since we are not using requestAnimationFrame and we haven't added the package as a dependency, removing this.